src/pages/resource/LinkPublicationPanel.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PublicationExpansionPanel from './PublicationExpansionPanel';
+import { TextField, Button } from '@material-ui/core';
```

Comment:
Material UI anbefaler path imports over named imports for å minimere build size: https://material-ui.com/guides/minimizing-bundle-size/#option-1